pkg/common/telemetry/names.go

```diff
@@ -305,6 +305,9 @@ const (
 	// Mode tags a bundle deletion mode
 	Mode = "mode"
 
+	// Network tags some network name (for example, "tcp", "udp")
```

Comment:
we only support TCP and UDS, insted of `for example` can we say here `Network tags some network name ("tcp", "udp")`